R/report.default.R

```diff
@@ -22,12 +22,12 @@
 #' @return An object of class [report()].
 #' @export
 report.default <- function(x, ...) {
-  # You can remove the following line once the functions below are implemented
-  stop(.error_message(x, "report()"), call. = FALSE)
+  # text <- report_text(x, ...)
+  # table <- report_table(x, ...)
+  # as.report(text = text, table = table, ...)
```

Comment:
There are so many changes like these where code is commented out. Also, the corresponding comment is not that helpful:

> # You can remove the following line once the functions below are implemented

If, at the moment, we don't need this code and this is just in case we add support for some things in the future, you should just remove this code. That future may never come and we don't need to carry the weight of this dead code.

Reply:
I was also surprised that every function in `report.default` is non-functional in the sense that each one of them start with a `stop()` error message. Lints said to not have code or comments after a `stop()`, so that's why I moved it above it (and commented to avoid errors) to try to change as few things as possible. The error you mention is simply an adaptation of the several similar messages present in the code. Given that it is systematic in this file, instead of removing only the commented code, should I simply delete the whole file since it does not seem to add anything? It's not only dead code, it's a dead file...

Original: https://github.com/easystats/report/blob/main/R/report.default.R

Reply:
@DominiqueMakowski Since you authored this file, can you please shed some light on what's going on here? Can we just delete this entire file?

Reply:
I think the idea here is just to inform the user to create an issue for regression models which are not supported. Since the default method will be triggered when no other method is found.

So we shouldn't delete the entire file, but we can delete all the commented code. But feel free to do that in a new PR.

Reply:
@IndrajeetPatil would you like me to delete only the commented code as you wrote, or also the (lengthy) original actual comments?